remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -331,8 +335,7 @@
                 }
             ],
             "dependsOn": [
-                "[resourceId('Microsoft.Web/serverfarms', variables('hostingPlanName'))]",
-                "[resourceId('Microsoft.Network/publicIPAddresses', variables('publicIPName'))]"
```

Comment:
why did we remove the publicipname?

Reply:
Actually this PR is what Devis added i just reverted my change back and added the port 443 fix. But i have tested it this change and don't think we need this dependsOn. Previously i was consuming the REMOTE_ENDPOINT value a little differently which gave inconsistent result. The way it is right now works great, so keeping this removed won't hurt us.